script.js

```diff
@@ -2,54 +2,81 @@
 // prepare DOM Elements
     const todoInput = document.querySelector('.todo-input')
     const ulList = document.querySelector('.todolist ul')
-    const deleteBtn = document.querySelector('.delete')
+    // const deleteBtn = document.querySelector('.delete')
     // const circle = document.querySelector('.circle')
+    // const check = document.querySelector('.fas fa-check')
     
     
     // This function adds new todo, after press enter. 
     const addNewTodo = (e) => {
         if (todoInput.value !== '' && e.key === 'Enter') {
```

Comment:
Add validation for the input value. Use trim to remove leading and trailing white spaces.

```
// Before
if (todoInput.value !== '' && e.key === 'Enter')

// After
if (todoInput.value.trim() !== '' && e.key === 'Enter')

```